src/main/java/com/example/demo/controller/UserController.java

```diff
@@ -0,0 +1,63 @@
+package com.example.demo.controller;
+
+import com.example.demo.dto.CardDTO;
+import com.example.demo.dto.UserDTO;
+import com.example.demo.service.CreditCardService;
+import com.example.demo.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import javax.validation.Valid;
+
+// Controller to User request management.
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+  @Autowired UserService userService;
+
+  @Autowired CreditCardService creditCardService;
```

Comment:
Segun este articulo que habia pasado Mati (https://springframework.guru/best-practices-for-dependency-injection-with-spring/), la mejor practica para inyectar beans seria directamente poniendolo como atributos final y con un @RequiredArgsConstructor

Reply:
genial voy a mirarlo!